### PR TITLE
Set contentType for database uploads

### PR DESCRIPTION
**Pull request by rneatherway**
### Merge / deployment checklist

- [ ] Confirm this change is backwards compatible with existing workflows.
- [ ] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [ ] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

File: src/database-upload.ts [hunk 1]
```diff
@@ -63,6 +63,7 @@ export async function uploadDatabases(
           data: payload,
           headers: {
             authorization: `token ${apiDetails.auth}`,
+            contentType: "application/zip",
```
Comment (rneatherway) [inline src/database-upload.ts:66]: I'm not 100% confident about the format of this line, so I'll test it using this branch.

Comment (aeisenberg) [reply src/database-upload.ts:66]: Looks right to me.

**Review by aeisenberg — APPROVED** [review]
Feel free to merge if this passes all of your tests.

**Comment by rneatherway** [comment]
Using the string literal as a header key worked :+1:

**Pull request merged**